docs/FAQ.md

```diff
@@ -57,6 +69,15 @@ We typically do not assign issues to anyone other than long-time contributors. I
    - Did you follow the pull request checklist?
    - Did you give your pull request a meaningful title?
 
+### How can I help review pull requests that are ready for quality assurance and approval?
+
+You can review PRs that need to be checked for quality assurance using [this page](https://github.com/freeCodeCamp/freeCodeCamp/pulls?q=is%3Aopen+is%3Apr+-label%3A%22status%3A+blocked%22+-label%3A%22status%3A+merge+conflict%22+status%3Asuccess+draft%3Afalse), which will have the following filters applied:
+
+- Ready for review (not-draft)
+- Do not have a merge conflict
+- Are not blocked
+- Have all status checks in green
```

Comment:
Thanks this is great, I think we should add something about how you can join the moderator's team here too. Because while we encourage reviews, we encourage new contributors to OSS "to be part of the team" and be involved.

There is a moderators handbook that you can use for cross linking.

Reply:
In my experience, actively encouraging non-maintainer reviews creates a lot of noise. I've seen in other orgs cases where approvals are given without understanding the changes (thus, approvals that should not have been there), or even approvals after a PR is merged.

Reply:
> add something about how you can join the moderator's team here too.

Well, I am not sure what that is. So, probably a nice idea to add 😅 

I am happy with a short section on helping QA, but suggest even less:
```suggestion
### How can I help review pull requests that are ready for quality assurance?

You can review PRs that need to be quality assured using [this page](https://github.com/freeCodeCamp/freeCodeCamp/pulls?q=is%3Aopen+is%3Apr+-label%3A%22status%3A+blocked%22+-label%3A%22status%3A+merge+conflict%22+status%3Asuccess+draft%3Afalse).
```

Reply:
I think @nhcarrigan is correct about non-maintainer reviews becoming noise. 

That said, I am in favour of coming up with some guidelines on how to contribute in a meaningful way to reviews. A path to join the moderator's team would be helpful IMHO. 

We always need more people on the @freeCodeCamp/moderators team because volunteers are busy, and we need all the help we can get.

Reply:
I agree with putting together a path for joining the moderation team.

Reply:
Do you have some recommendations, considering your experience with communities outside of freecodecamp? (we can hash the details on this PR or a dedicated one once we settle on something offline, if you prefer)

Reply:
In the past we've had a very "don't call us, we'll call you" approach to becoming a moderator because we want to pull from community leaders. It does seem to me that ideally we would want moderators to have been consistent members of the platform for a certain amount of time. I think that being familiar and comfortable with our community is really important.
On the forum it's pretty easy to look at a history of their interactions with other campers, but I don't know much about what criteria you could use to vet a Discord member.

Reply:
Hi, I’ll chime in a bit on this. I think there are discord bots that will measure and track users’ message history and activity. That data can be useful to what we’re trying to achieve

Reply:
@raisedadead realistically, the path would primarily reflect how to engage with our platforms in meaningful ways.

As @ArielLeslie mentioned, our process has always been to identify and reach out to active members and invite them to join the team - so I would argue that the best "roadmap" would focus on how to contribute meaningfully on the forum/discord/chat/github, etc.

Reply:
Thanks for those great insights, Ariel and Parth. Appreciate the feedback. 

Parth, we already have a dashboard to get metrics from multiple communities (GitHub, forum, Discord, and more) with our Orbit integration (essentially we have a way to identify folks who are most active and where etc.), that's not what we need though right now.

At the end of the day, we would still follow the "don't call us, we'll call you" approach, that Ariel correctly has pointed out.

The idea I guess from this PR is: 

We want to document, how as a newcomer to freeCodeCamp's community, you can be on that path of establishing your credibility (over a certain period of time). Documenting the process in verbiage that talks about the recommended way to approach being a moderator should point interested folks in the right direction, IMHO.

Suggestions about what we should include in such documentation (for newcomers) would be awesome.

Reply:
So maybe more of some suggestions on how best to offer help and leadership? Things like tips on how to answer questions effectively, when to flag posts for moderation, using encouraging/supportive language, cross-cultural communication, avoiding online drama...
Maybe an external News article that could be linked in a 'so you wanna moderate' section?

Reply:
> Things like tips on how to answer questions effectively, when to flag posts for moderation, using encouraging/supportive language, cross-cultural communication, avoiding online drama...

I like these ideas and think we would definitely benefit from having them. I'm not sure if a news article is better than having it within the documentation, only because it is a bit harder to edit/maintain a news article (fewer people have access to change the content).

I think we should probably create an issue to discuss + brainstorm content, so we don't bog down this PR too much.

Reply:
> I think we should probably create an issue to discuss + brainstorm content, so we don't bog down this PR too much.

I agree.

@vincentchen137 thanks for the patience, do you mind removing this section while we discuss this in a separate thread. I think the rest of the PR can go in.

Reply:
I created a thread, lets move our discussion there instead.

Reply:
@raisedadead yes of course! I'll send a commit without that section shortly. 

Sorry if I added this section unnecessarily, I had found your pinned thread with this information on the forums and thought it would be helpful in the FAQ section to entice more experience OSS contributors to join the community. I am glad, however, that it's sparked a conversation on how contributors could get more involved in helping moderators directly and perhaps even becoming one.

Reply:
Thanks @vincentchen137 - we are glad as well we are having this conversation as well. Cheers.